src/environment/generators/random_grid_generator.py

```diff
@@ -2,29 +2,29 @@
 from typing import Tuple, Union
 
 from environment.generators.generator import GridGenerator
-from environment.types import Grid, Location
+from environment.types import Grid, Room
 
 
 class RandomGridGenerator(GridGenerator):
 
     def __init__(self, dungeon_size: int, seed: int) -> None:
         super().__init__(dungeon_size, seed)
 
-    def generate(self) -> Tuple[Grid, Location]:
+    def generate(self) -> Tuple[Grid, Room]:
```

Comment:
tu je zle napisany typing hint, co sa vracia. v abstraktnej triede `Generator` je to opacne, ja s tym pocitam teda opacne. opravim to.

Reply:
ok, pokecame v sobotu